src/components/social-panel/follow-panel.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { isBrowser } from '../util/is-browser';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { IconRSS } from '../icons/social/rss';
+import { SimpleLink } from '../markdown/custom-components';
+
+const RSS_FEED_URL = '/blog/rss.xml';
+
+export interface ShareProps {
+  title: string;
+}
+
+const FollowPanelContainer = styled.div`
+  color: #668cff;
+  margin-top: 48px;
+  font-weight: bold;
+  font-size: 16px;
+
+  ${up('md')} {
+    margin-top: 0px;
+    min-width: 240px;
+  }
+`;
+
+export const PanelText = styled.p`
+  font-weight: bold;
+  font-size: 16px;
+`;
+
+const SocialLinks = styled.ul`
+  all: unset;
+  order: 2;
+
+  ${up('md')} {
+    order: 1;
+    margin-bottom: 20px;
+  }
+`;
+
+const SocialLinkItem = styled.li`
+  display: inline-block;
+  margin-right: 16px;
+  cursor: pointer;
+  &:last-of-type {
+    margin-right: 0;
+  }
+`;
+
+export const FollowPanel: React.FC = () => {
+  const [rssUrl, setRssUrl] = useState('');
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    if (isBrowser()) {
+      const rssURL = window.location.origin.concat(RSS_FEED_URL);
+      setRssUrl(rssURL);
+    }
+  });
+
+  return (
+    <FollowPanelContainer>
+      <PanelText>{t('blog.follow')}</PanelText>
+      <SocialLinks>
+        <SocialLinkItem data-testid="rss-feed">
+          <SimpleLink href={rssUrl}>
+            <IconRSS />
```

Comment:
That icon looks good on small and large devices👍

![Screen Shot 2021-08-31 at 15 34 41](https://user-images.githubusercontent.com/1701755/131511959-28a3f0a8-6662-41da-b1c9-46c9f76bb7f9.png)

![Screen Shot 2021-08-31 at 15 34 19](https://user-images.githubusercontent.com/1701755/131511900-9e3c92d1-0849-4b07-8d1e-f20073c30305.png)